tests/test_taskgroups.py

```diff
@@ -1837,3 +1837,14 @@ async def test_patched_asyncio_task(monkeypatch: MonkeyPatch) -> None:
     )
     async with create_task_group() as tg:
         tg.start_soon(sleep, 0)
+
+
+@pytest.mark.parametrize("anyio_backend", asyncio_params)
```

Comment:
This should pass on trio and asyncio, we want to make sure the behavior is the same

Reply:
Fair. This test seems to have been added since I took a look at this changeset.

Reply:
It already passes on trio so I'll just remove the marks here then.